### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by adrian-ponce-wolox**
## Summary ##
Navigation implementation

* Show tabs “Chats”, “Grupos” and “Contactos” y navigate between them
* Add contacts/groups button

navigation

screen shot 2017-10-03 at 6 18 21 pm
screen shot 2017-10-03 at 6 18 18 pm
screen shot 2017-10-03 at 6 18 14 pm

### CODE REVIEW

File: wchat/screens/GroupList/components/GroupCell/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { AppRegistry, Text, Image, View } from 'react-native';
+import styles from './styles.js'
+
+export default class GroupCell extends Component {
+  render() {
+    console.log("this.props.name" + this.props.name)
```
Comment (IgnacioAbadie) [inline wchat/screens/GroupList/components/GroupCell/index.js]: remove `console.log`

File: wchat/screens/GroupList/components/GroupCell/styles.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const styles = StyleSheet.create({
+  container: {
+   flex: 1,
+   flexDirection: 'row',
+   marginVertical: 12,
+  },
+  image: {
+    height: 80,
+    width: 80,
```
Comment (IgnacioAbadie) [inline wchat/screens/GroupList/components/GroupCell/styles.js]: extract to const

File: wchat/screens/Home/styles.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from 'react-native';
+import { mainBackgroundColor } from '../../shared/utils/colors.js'
+
+const styles = StyleSheet.create({
+  tabbar: {
```
Comment (IgnacioAbadie) [inline wchat/screens/Home/styles.js]: indentation

File: wchat/screens/Home/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Text } from 'react-native';
+import { TabNavigator } from "react-navigation";
+import { mainColor } from '../../shared/utils/colors.js'
+import ChatList from '../ChatList/index.js'
+import ContactList from '../ContactList/index.js'
+import GroupList from '../GroupList/index.js'
+import styles from './styles.js'
+
+const MainScreenNavigator = TabNavigator({
```
Comment (IgnacioAbadie) [inline wchat/screens/Home/index.js]: move tabNavigator to a screen.js file

File: wchat/shared/utils/colors.js [hunk 5]
```diff
@@ -1,2 +1,6 @@
 export const messageColor = '#999';
 export const separatorColor = '#CCC';
+export const mainBackgroundColor = 'white';
```
Comment (IgnacioAbadie) [inline wchat/shared/utils/colors.js]: use `#FFF`

File: wchat/screens/GroupList/layout.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { AppRegistry, FlatList, View } from 'react-native';
+import GroupCell from './components/GroupCell/index.js'
+import AddButton from '../shared/components/AddButton/index.js'
+import add_group_icon from '../../shared/assets/add-group.png'
+import styles from './styles.js'
+
+export default class GroupList extends Component {
```
Comment (IgnacioAbadie) [inline wchat/screens/GroupList/layout.js]: Chat List and Group List are identicals, can you extract here something?

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]

**Comment by adrian-ponce-wolox** [comment]
Updated screens:
screen shot 2017-10-04 at 3 21 51 pm
screen shot 2017-10-04 at 3 21 45 pm

File: wchat/screens/navigationOptions.js [hunk 7]
```diff
@@ -9,3 +15,36 @@ export function homeOptions() {
...
+        style={[styles.icon, {tintColor: tintColor}]} />
+      );
+    }
+  }
+}
+
+export function groupsOptions(){
+  return {
+    tabBarLabel: GROUPS,
+    tabBarIcon: ({ tintColor }) => {
```
Comment (IgnacioAbadie) [inline wchat/screens/navigationOptions.js]: can extract this function like a const to avoid repeating code.

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]
Just a little change and ready :D

**Review by IgnacioAbadie — APPROVED** [review]

**Pull request merged**